code/src/components/QuestionTwo.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable jsx-a11y/no-noninteractive-tabindex */
+// eslint-disable-next-line no-unused-vars
+import React from 'react';
+
+// eslint-disable-next-line no-unused-vars, max-len
+const QuestionTwo = ({ questionTwo, setQuestionTwo, workAlternative, setWorkAlternative, onStepChange, onInputChange }) => {
```

Comment:
I do not think these two props are needed `questionTwo, setQuestionTwo` as they are defined in the Survey part.